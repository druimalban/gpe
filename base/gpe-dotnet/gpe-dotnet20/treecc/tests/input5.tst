// test comments

/* /* */ x

/*
// jkjkjkjkjk
i
*/

/**/

/* ***************************/
/****************************/

/*
*/

/*

*/

/* contains EOF
