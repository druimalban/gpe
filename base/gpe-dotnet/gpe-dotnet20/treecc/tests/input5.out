line 3: identifier (len = 1): x
input5.tst:22: end of file inside comment
